orangecontrib/text/widgets/owimportdocuments.py

```diff
@@ -110,15 +113,21 @@ class Outputs:
 
     class Warning(widget.OWWidget.Warning):
         read_error = widget.Msg("{} couldn't be read.")
+        no_tokens = widget.Msg("POS tags can't be used without lemmas.")
```

Comment:
It seems this one is not used. Even if I uncheck lemmas I do not get this error.